src/screens/ContactScreen/ContactScreen.tsx

```diff
@@ -1,32 +1,93 @@
 import React from 'react';
-import { Text } from 'react-native';
-import { SpacerInline } from '@elements';
-import InfoScreen from '../InfoScreen';
-import styles from './ContactScreen.styles';
+import {
+	ScrollView,
+	Text,
+	TouchableHighlight,
+	View,
+} from 'react-native';
+import { Linking } from 'expo';
+import {
+	ContentHeader,
+	NavBar,
+	SpacerInline,
+	Icon,
+} from '@elements';
+import { IconName } from '@elements/Icon';
+import styles, { ListItem } from './ContactScreen.styles';
+
+const contactList: Array<{
+	title: string;
+	message: string;
+	link: string;
+	iconName: IconName;
+}> = [
+	{
+		title: 'website',
+		message: 'www.bananaapp.org',
+		link: 'https://www.bananaapp.org',
+		iconName: 'website',
+	},
+	{
+		title: 'email',
+		message: 'info@bananapp.org',
+		// TODO: Supposedly this link works on real devices, not simulators. Verify.
+		link: 'mailto:info@bananapp.org',
+		iconName: 'email',
+	},
+	{
+		title: 'facebook',
+		message: 'Connect with us on Facebook!',
+		// TODO: Fix this navigation
```

Comment:
Like the `TODO` states, this link is broken. Acceptable bug for now?

Reply:
I'll add a TODO and task to update these links! From the little research I did, it might take a little effort to get this FB app link to work on all devices

Reply:
ooh okay, future goals! woo!